client/src/components/boxViewContainer.js

```diff
@@ -8,7 +8,7 @@ export default class BoxViewContainer extends React.Component {
   state = {
     box: "",
     voters: [],
-    votersAmount: "",
+    otersAmount: "",
```

Comment:
Not sure why voters was changed to oters :thinking:

Reply:
accidentally removed i noticed it after push will fix it !